services/postprocessing/README.md

```diff
@@ -20,12 +20,25 @@ When postprocessing has been enabled, configuring any postprocessing step will r
 
 ## Postprocessing Steps
 
-As of now, `ocis` allows two different postprocessing steps to be enabled via an environment variable.
+`ocis` allows setting the order and amount of postprocessing steps via the `POSTPROCESSING_STEPS` envvar. It expects a comma seperated list of steps that should be executed. Known to the system are `virusscan` and `delay`. It is allowed to set custom steps.
```

Comment:
```suggestion
The postporcessing service is individually configurable. This is achieved by allowing a list of postprocessing steps that are processed in order of their appearance in the `POSTPROCESSING_STEPS` envvar. This envvar expects a comma separated list of steps that will be executed. Currently known steps to the system are `virusscan` and `delay`. It is allowed to set custom steps.
```
Not clear what the sentence ` It is allowed to set custom steps.` means or what the impact is.

Reply:
It is explained in more detail in `Custom Postprocessing Steps` section.

Reply:
I rephrased:
`It is allowed to set custom steps.` --> `Custom steps can be added but need an existing target for processing.`